Documentation/benchmarks/etcd-2-2-0-benchmarks.md

```diff
@@ -0,0 +1,67 @@
+# Physical Machines
+
+GCE n1-highcpu-2 machine type
+
+- 1x dedicated local SSD mounted as etcd data directory
+- 1x dedicated slow disk for the OS
+- 1.8 GB memory
+- 2x CPUs
+
+# etcd Cluster
+
+3 etcd 2.2.0 members, each runs on a single machine.
+
+Detailed versions:
+
+```
+etcd Version: 2.2.0
+Git SHA: e4561dd
+Go Version: go1.5
+Go OS/Arch: linux/amd64
+```
+
+# Testing
+
+Bootstrap another machine and use benchmark tool [boom](https://github.com/rakyll/boom) with customized patch to send requests to each etcd member. Check [here](../../hack/benchmark/) for instructions.
+
+The performance is the result of running benchmark script one round. In the testing cluster for this benchmark, the result of multiple rounds could be different up to 20%. An even bigger difference 30% was observed in clusters with the same configuration.
```

Comment:
Mm, I think benchmark results from a single run here are pretty useless when there's this much variance. Can we do the average/stdev thing?